frontend/yarn.lock

```diff
@@ -5,6 +5,7 @@
 "3d-view-controls@^2.2.0":
   version "2.2.2"
   resolved "https://registry.yarnpkg.com/3d-view-controls/-/3d-view-controls-2.2.2.tgz#7173fc197e9e7e4dbc63213438467047dbc84fa2"
+  integrity sha512-WL0u3PN41lEx/4qvKqV6bJlweUYoW18FXMshW/qHb41AVdZxDReLoJNGYsI7x6jf9bYelEF62BJPQmO7yEnG2w==
```

Comment:
For now, can you use an older version of yarn until we are able to update to the latest? Otherwise PRs will constantly add and remove the `integrity` lines from yarn.lock depending on the yarn version.

Reply:
What yarn version are we using?  README says "yarn >= 1.3.2"